src/cluster.c

```diff
@@ -4573,6 +4576,9 @@ void migrateCommand(redisClient *c) {
             copy = 1;
         } else if (!strcasecmp(c->argv[j]->ptr,"replace")) {
             replace = 1;
+        } else if (!auth) {
```

Comment:
it meas that if you use migrate command like this "migrate 127.0.0.1 40000 hash_resume 0 1000000 copy replace password1 abcdefg"  and then the key word "password1" is considered as a password but the word "abcdefg" is a wrong argument.